providers/class.two-factor-email.php

```diff
@@ -51,7 +51,7 @@ function authentication_page( $user ) {
 		<p><?php esc_html_e( 'A verification code has been sent to the email address associated with your account.', 'two-factor' ); ?></p>
 		<p>
 			<label for="authcode"><?php esc_html_e( 'Verification Code:' ); ?></label>
-			<input type="tel" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
+			<input type="text" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
```

Comment:
Using `input[type="text"]` with the numeric pattern [should be enough to trigger the numeric keypad](http://bradfrost.com/blog/mobile/better-numerical-inputs-for-mobile-forms/) (at the bottom of the article Brad points out the pattern attribute is enough to trigger the behavior), at least on iOS, though it would be beneficial to [do further device testing using the "Text pattern=[0-9]" input here](http://bradfrost.com/demo/tel/).

Screenshot of a text input with the pattern attribute in Xcode's iOS simulator (iPhone 6, iOS 8.2):

![ios simulator screen shot jul 13 2015 5 27 45 pm](https://cloud.githubusercontent.com/assets/233836/8661359/074de6fc-2985-11e5-8b10-b92a26f8c0fc.png)